datasetinsights/configs/faster_rcnn_synthetic.yaml

```diff
@@ -7,6 +7,7 @@ train:
     name: SynDetection2D
     args:
       run_execution_id: zMvGEV0
+      load_local: False
```

Comment:
Why do we need this option for users? Can we simply check if the dataset downloaded first? If not, we can download the synthetic data.